src/Aquarius.ONE.ClientSDK/Common/Configuration/Graph/GraphConfigurationData.cs

```diff
@@ -1,22 +0,0 @@
-﻿using System.Collections.Generic;
-
-namespace ONE.Common.Configuration.Graph
-{
-    public class GraphConfigurationData

```

Comment:
There is another copy of this class (unused) that can be removed to avoid confusion: 
https://github.com/AquaticInformatics/aquarius-one-sdk/blob/9440b70c819401a8cc2bf1249a7b57bd941d5cb1/src/Aquarius.ONE.ClientSDK/Common/Configuration/GraphConfigurationData.cs#L5-L18

Reply:
I am going to remove the documentation here.  It should now be located: https://aquaticinformatics.atlassian.net/wiki/spaces/AOS/overview

Reply:
Good catch.  Removed